lib/_filters/convertColor.js

```diff
@@ -0,0 +1,24 @@
+let convert = require('color-convert');
```

Comment:
I think we should look to use ES modules going forward because of support in Eleventy 3.0.0